.circleci/config.yml

```diff
@@ -77,6 +77,22 @@ jobs:
       - store_artifacts:
           path: ./test-results
 
+  script-test:
```

Comment:
Perhaps something like `test-drupal-cache` might be more descriptive?

Reply:
i was thinking there are other critical scripts that might be added to this test step, but i'm not sure what those are yet. maybe its better to have more specific name now and change later if additional scripts are added?